modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/Validation.js

```diff
@@ -50,6 +50,7 @@ const Validation = ({
 	validation,
 	validations,
 	visible,
+	...otherProps
```

Comment:
I expect all parameters related to the `InputMask` feature (`append`, `inputMaskFormat`, `symbols`, ...).
I didn’t want to be more explicit because these parameters are specific to the `Numeric` field, and we also have the `ValidationDate` component. What do you think in this case?

Reply:
Oh! It makes sense, @carolmariaabb. Thanks for clarifying.